app/views/services/select-org-type.njk

```diff
@@ -78,7 +78,9 @@
 
   <script>
     document.getElementById('submit-button').addEventListener('click', function () {
-      disableButtonAndShowSpinner()
+      if (document.getElementById('org-type-local').checked) {
```

Comment:
is it necessary to perform this check? If so, you probably also want to account for the user not selecting anything as that will also result in a page load

Reply:
The button doesn't necessarily show e.g. because you can't request a Stripe test account if your service is live or you've already submitted a request (under the old system where it was done manually). The code for manual creation can probably be removed though